config/openshift-customizations.yaml

```diff
@@ -18,3 +18,6 @@ releases:
       - "^pull-ci-openshift-.*-(master|main).*-e2e-.*"
       - "^pull-ci-operator-framework.*-(master|main).*-e2e-.*"
       - "^pull-ci-openshift-online.*-(master|main).*-(verify|unit).*"
+  aro-integration:
+      jobs:
+        periodic-ci-Azure-ARO-HCP-main-periodic-create-aro-hcp-in-integration: true
```

Comment:
aside from presubmits, jobs of interest are pulled from the release-controller. aro-hcp doesn't have a release-controller; so these need to be configured manually like presubmits.